forge-gui/res/cardsfolder/upcoming/arteeoh_dread_scavenger.txt

```diff
@@ -0,0 +1,13 @@
+Name:Arteeoh, Dread Scavenger
+ManaCost:1 B G U
+Types:Legendary Artifact Creature Robot
+PT:3/3
+K:Flying
+K:Deathtouch
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | TriggerZones$ Battlefield | Execute$ TrigExchange | TriggerDescription$ Whenever NICKNAME deals combat damage to a player, you may exchange control of two other target artifacts. When you do, create a token that's a copy of target artifact you don't control, except it’s a 1/1 green Squirrel creature token in addition to its other colors and types.
+SVar:TrigExchange:DB$ ExchangeControl | RememberExchanged$ True | ValidTgts$ Artifact.Other | TargetMin$ 2 | TargetMax$ 2 | TgtPrompt$ Choose two other target artifacts | Optional$ True | AILogic$ TrigTwoTargets | StackDescription$ SpellDescription | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True | SubAbility$ TrigImmediateTrig
```

Comment:
Now you're doing it too early, look at the order ;)
(there's no way the Conditions can be true now)